Edit DPTA to work better for mac

hnn0003 opened the pull request:
- Change the dpta to support non-Windows paths.
- Get rid of support for closing the browsers at the end of the dpta tests--this is done better & with less errors in the YAML running the automated tests.
- Change saved csv file to put the view name in quotes, the same as the iModel name & view flags & other things (this is done to ensure that view names with commas as part of the view name won't mess up the csv file).
- Edit the package.json to support "npm run test" and "npm run test:electron", instead of just "npm run start" and "npm run start:electron".

pmconne commented:
Please describe the changes in the description field.
Please add information to the README re: running the tests on a mac and what the "no_debug" option does.

hnn0003 commented:
> Please describe the changes in the description field.
> Please add information to the README re: running the tests on a mac and what the "no_debug" option does.

@pmconne I've updated the README, and I think I've added the description? I added a description, at least. Not totally sure if it's actually in the description field...

pmconne commented:
@Mergifyio update
